toreqif.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 mkdir -p reqif
 strictdoc export --formats=reqif-sdoc --project-title "Heavy Duty Vehicle Cybersecurity Requirements (HD VCR)" --output-dir . *.sdoc
-mv reqif/output.reqif reqif/vcr-experiment.reqif
+mv reqif/output.reqif docs/html/vcr-experiment.reqif
```

Comment:
@jmaag15 this line needs to execute in CI before https://nmfta-repo.github.io/vcr-experiment/vcr-experiment.reqif would be live

Reply:
Ahh okay